tests/behaviour/test_advanced_identity.py

```diff
@@ -896,12 +914,19 @@ def test_add_a_control_delegation_between_2_existing_registered_identities():
 
 
 @scenario('advanced_identity_api.feature',
-          'Add a control delegation proof (created by an other registered identity) to a document',
+          'Add a control delegation did proof (created by an other registered identity) to a document',
           features_base_dir=FEATURES)
 def test_add_a_control_delegation_proof_from_an_other_registered_identity_to_a_document():
     pass
 
 
+@scenario('advanced_identity_api.feature',
+          'Add a control delegation generic proof (created by an other registered identity) to a document',
+          features_base_dir=FEATURES)
+def test_add_a_control_delegation_reusable_proof_from_an_other_registered_identity_to_a_document():
```

Comment:
are these here to just add doc step for bdd?

Reply:
It is there for pytest to detect the new BDD scenario